scalafix/rules/src/main/scala/rsc/rules/RscCompat.scala

```diff
@@ -136,6 +162,9 @@ case class RscCompat(legacyIndex: SemanticdbIndex, config: RscCompatConfig)
               .get
           }
           buf += RewriteDefn(env, before, name, after, body, parens = false)
+          buf ++= targetsForPolymorphicDefaultParams(env, name, paramss)
+        case defn @ Defn.Def(_, name, _, paramss, _, _) =>
+          buf ++= targetsForPolymorphicDefaultParams(env, name, paramss)
```

Comment:
Other trees can also have default parameters, e.g. constructors, macros and abstract methods (i.e. Decl.Def).